src/Tribe/Capabilities.php

```diff
@@ -1,9 +1,40 @@
 <?php
 
 class Tribe__Events__Capabilities {
-	public $set_initial_caps = false;
-	private $cap_aliases = array(
-		'editor' => array( // full permissions to a post type
+
+	/**
+	 * The transient key for delayed capabilities updates.
+	 *
+	 * @since TBD
+	 *
+	 * @var string
+	 */
+	protected $key_delayed_capabilities_update = '_tribe_events_delayed_capabilities_update';
+
+	/**
+	 * An array of roles to update capabilities.
+	 *
+	 * @since TBD
+	 *
+	 * @var array
+	 */
+	protected $roles = [
+		'administrator',
+		'editor',
+		'author',
+		'contributor',
+		'subscriber'
```

Comment:
* :no_entry_sign: Each array item in a multi-line array declaration must end in a comma
* :no_entry_sign: Comma required after last value in array declaration

<sub>hash: d6467873f0ea426fa74082b2fcd9d41f</sub>